models/users.js

```diff
@@ -178,6 +178,33 @@ const fetchUsers = async (query) => {
   }
 };
 
+const fetchAllUsers = async () => {
+  try {
+    const dbQuery = userModel;
```

Comment:
`userModel` is a model why are we assigning it to a query variable?

Reply:
This defines the model of the query we want to be fetched.